editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5798,12 +5798,8 @@ void CanvasItemEditorViewport::_create_nodes(Node *parent, Node *child, String &
 	}
 
 	String node_class = child->get_class();
```

Comment:
Would it be possible to use `Object::cast_to` to find the node types instead of relying on strings? This would have prevented this bug if it was the case here.

Reply:
Yes, I can do that.